ansible/roles/providers/tasks/azure-pip.yml

```diff
@@ -12,6 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 ---
+- name: install pip
+  package:
+    name: python3-pip
+    state: present
+  when:
+    - ansible_os_family == "Debian"
```

Comment:
This is technically redundant given that `packages` should run first, but it bring in parity with `ansible/roles/providers/tasks/aws.yml` which also has an ensure here.